Library/Homebrew/descriptions.rb

```diff
@@ -123,9 +124,21 @@ def initialize(descriptions)
   # print them.
   def print
     blank = "#{Tty.yellow}[no description]#{Tty.reset}"
-    @descriptions.keys.sort.each do |name|
-      description = @descriptions[name] || blank
-      puts "#{Tty.white}#{name}:#{Tty.reset} #{description}"
+    @descriptions.keys.sort.each do |full_name|
+      short_name = short_names[full_name]
+      printed_name = unique_short_names.include?(short_name) ? short_name : full_name
```

Comment:
How could `short_name` not be in `unique_short_names` since the latter contains all values from `short_names`?